src/Analyzers/CSharp/Tests/UseCollectionInitializer/UseCollectionInitializerTests_CollectionExpression.cs

```diff
@@ -2290,11 +2490,1366 @@ class C
             {
                 void M()
                 {
-                    List<int> c = [1,
+                    List<int> c =
+                    [
+                        1,
                         2,
                     ];
                 }
             }
             """);
     }
+
+    [Fact]
+    public async Task TestReplacementLocation_NoElements_ExistingInitializer1()
```

Comment:
thus begins a huge swath of tests created to validate a ton of whitespace/newline trivia scenarios.